about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/index.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>About</title>
+</head>
+<body>
+    <div class="container">
+        <a class="home" href="./index.html"><h3 class="name">Okewusi Oluwapelumi</h3></a>
+        <p class="title">Front end developer and designer</p>
+        <div class="main-container">
+            <div class="about-me">
+                <h3>About Me</h3>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi autem commodi omnis deserunt, atque delectus ratione? Ad ipsum, quis esse corrupti ut accusamus odit quam natus. Obcaecati non cupiditate modi? Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt non sequi debitis corrupti nisi obcaecati accusamus quisquam saepe. Illo, eius animi accusamus vero quidem natus? Ratione sed laboriosam suscipit tempore.</p>
+            </div>
+            <div class="bottom">
+                <div class="picture"></div>
```

Comment:
kindly use your picture here

Reply:
I will put one random one, for now, I don't really have a professional one at the moment.